tensorboard/webapp/plugins/npmi/testing/index.ts

```diff
@@ -58,3 +58,98 @@ export function appStateFromNpmiState(metricsState?: NpmiState): State {
     [NPMI_FEATURE_KEY]: metricsState || createNpmiState(),
   };
 }
+
+export function getSampleAnnotationData() {
```

Comment:
We tend to use prefix like `create` for creating a fixture. Similarly, we use prefix `build` for fixture builder.